### ISSUE TITLE
Make search compatible with screen readers

### ISSUE THREAD

**Issue by plexish** [StampyAI/stampy-ui#13]

**Comment by tayler6000** [comment]
@Aprillion do you know if this has been done already?  If so, will you close this issue?

**Comment by Aprillion** [comment]
I think @mruwnik made some improvements in keyboard navigation of search, but not sure if anyone actually tested the site with actual screen reader yet...

@plexish - it might be a good idea to find some accessibility testing volunteer to make a review of the site, if there are some  low hanging improvements we should make

**Comment by mruwnik** [comment]
This. There were a few incidental improvements, but until someone with an actual reader checks it, this is pretty much blocked

**Comment by plexish** [comment]
Okay, I'll mention this in my next volunteer call.

**Comment by tayler6000** [comment]
I tried out ui.stampy.ai and aisafety.info with iOS VoiceOver and I think there is definitely some improvements we could make.  But at least for the search bar, that seemed totally fine to me.  The only thing I noticed was that with the plus in `＋ None of these: Request an answer to my exact question above` was pronounced in a Japanese accent on my phone, so I don't know if we're using some weird unicode + and not ASCII or if my phone was just confused because my phone can do both English and Japanese.
